api/src/main/java/org/apache/iceberg/Schema.java

```diff
@@ -507,4 +538,49 @@ public String toString() {
                 .map(this::identifierFieldToString)
                 .collect(Collectors.toList())));
   }
+
+  /**
+   * All ids of metadata fields are reassigned.
```

Comment:
Not sure what this line means?

Reply:
Sorry, tried to make it clear, but feel free to suggest if still not.